rockspecs/tranquility-dev-1.rockspec

```diff
@@ -1,44 +1,48 @@
 package = "tranquility"
 version = "dev-1"
 source = {
-    url = "git+ssh://git@github.com/XiNNiW/tranquility.git"
+	url = "git+ssh://git@github.com/XiNNiW/tranquility.git",
 }
 description = {
-    summary = "a port of the tidalcycles pattern language to lua",
-    detailed = [[
+	summary = "a port of the tidalcycles pattern language to lua",
+	detailed = [[
 Tranquility is an experimental port of the livecoding music language [Tidalcycles](http://tidalcycles.org/) to the lua programming language.
 This project follows in the footsteps of [vortex](https://github.com/tidalcycles/vortex) and [strudel](https://strudel.tidalcycles.org).
 For me the main purpose of this project is to learn about how to implement a livecoding language.]],
-    homepage = "https://github.com/XiNNiW/tranquility",
-    license = "GPL3"
+	homepage = "https://github.com/XiNNiW/tranquility",
+	license = "GPL3",
 }
 dependencies = {
-    "lua >= 5.1",
-    "luasec >= 1.2.0-1",
-    "losc >= 1.0.1-1",
-    "luasocket >= 3.1.0-1",
-    "abletonlink >= 1.0.0-1"
+	"lua >= 5.1",
+	"luasec >= 1.2.0-1",
+	"losc >= 1.0.1-1",
```

Comment:
Is lpeg built into lua or does it need to go in the dependencies? If its not built in, would you please add lpeg to the lists of dependencies?

Reply:
yes you r right, I should add it.